packages/atomic/src/components/atomic-search-box-v1/atomic-search-box.pcss

```diff
@@ -0,0 +1,53 @@
+@import '../../global/global.pcss';
+
+.container {
```

Comment:
I'm not sure I understand the logic we are trying to apply for tailwind...

My gut feeling would be that we try to follow this pattern:

* If we see a set of tailwind classes that are repeated across multiple components, then we extract an `@apply` directive, and reuse that for all components that need it. So there would, in theory (?) be no such `@apply` directive in a specific `pcss` file for a specific component: only in shared/global folders.

* There is also the `@layer` directive that exists in tailwind, that we might want to utilize, since it has an impact on css purging.

* Otherwise, there would be no reason to extract "classic" css classes like this, with an `@apply`. And that we would just have all the tailwind utilities directly in the `.jsx` code. Because otherwise, it does seems like trying to fight a bit against tailwind ?

What's your opinion about this ?

Reply:
I agree with this pattern, because of the JIT mode I mainly used the pcss directly but I can move those classes in back the TSX. 

I'll investigate the `@layer` for the buttons stuff.